CONTRIBUTING.md

```diff
@@ -16,25 +16,25 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 ## Submit an Issue
 
 if you wish to contribute to GCToolKit we would kindly ask that you submit an issue to the issue tracker. Doing so will help with the management of the project.
- 
+
 ## Build
 
-The build requires JDK 11, and uses the Maven wrapper (`mvnw`) to help ensure reproducible builds and so we don't force you to change your system Maven install. If you prefer to build with your local Maven installation, make sure the version matches the one in the project's [.mvn/wrapper/maven-wrapper.properties](https://github.com/microsoft/gctoolkit/blob/main/.mvn/wrapper/maven-wrapper.properties) file.
+The build requires JDK 11 (you can also use JDK 17 or 21), and uses the Maven wrapper (`mvnw`) to help ensure reproducible builds and so we don't force you to change your system Maven install. If you prefer to build with your local Maven installation, make sure the version matches the one in the project's [.mvn/wrapper/maven-wrapper.properties](https://github.com/microsoft/gctoolkit/blob/main/.mvn/wrapper/maven-wrapper.properties) file.
 
-* `mvnw clean` - remove build artifacts
-* `mvnw compile` - compile the source code
+* `./mvnw clean` - remove build artifacts
+* `./mvnw compile <11|17|21>` - compile the source code. WARN you must set `11`, `17` or `21` for the release version.
```

Comment:
You don't need `-Dmaven.compiler.release` here? And why can't it default to 21 if maven.compiler.release is not set? 

Maybe what's needed is a github action profile that allows the matrix version. If I'm building this myself, I should be able to do a simple `./mvnw package` without all of the ceremony.

Reply:
Good point. In order to support the GH Matrix builds I had to be able to pass in the `maven.compiler.release` property but it would be better with a default somehow.  I'll see if I can figure out how to do that.

Is 21 the default we want? IIRC we have users that have built Java 11 based integrations with this library and we have not used any Java 12+ constructs in the source code.

Reply:
OK, I can set the default property in the pom.xml file and it can be overridden by the GH Action matrix build, I've pushed a fix for the pom file and docs